src/views/portals/utils/DashboardImagesGallery.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div class="box">
+    <h4 class="title is-3">
+      <router-link to="images">
+        <icon-image />
+        {{ $gettext('images') | uppercaseFirstLetter }}
+      </router-link>
+    </h4>
+    <div class="images-container">
+      <loading-notification :promise="imagesPromise" />
+      <gallery v-if="images != null" :images="images.documents" />
+    </div>
+    <hr />
+    <h6 class="title is-6 has-text-centered">
+      <router-link to="images">
+        <span v-translate>See more</span>
+      </router-link>
+    </h6>
```

Comment:
Styling differs from `See more` link in other boxes (not black when not hovered)